### PR TITLE
feat(FI-967) refactor Ledgers and tokens documentation

### PR DESCRIPTION
**Pull request by NikolasHai**
This MR introduces some refactoring to the Ledgers and tokens documentation. 
The branch used for this MR serves as an intermediary for other branches that have been merged into this branch. This was done to guarantee atomic updating of the documentation as well as reviewability of MRs. 

The details of the changes are discussed [here](https://docs.google.com/document/d/1eUnw7ki0Z-mOVu098AkT5Ih4_fnV3d-jIlX6xsDfFI0/edit)

### CODE REVIEW

File: docs/developer-docs/integrations/icrc-1/icrc1-ledger-setup.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ICRC-1 ledger setup
 
 ## Overview
-This guide will show you how to deploy an ICRC-1 ledger locally and on mainnet, how to interact with the ICRC-1 ledger and how to test it. ICRC-1 is a token standard which you can read up on [here](https://github.com/dfinity/ICRC-1). 
+This guide will show you how to deploy an ICRC-1 ledger locally and on mainnet, how to interact with the ICRC-1 ledger and how to test it. ICRC-1 is a token standard which you can read up on [here](https://github.com/dfinity/ICRC-1). Thus, deploying a new token that is ICRC-1 compatible is synonymous with deploying an ICRC-1 ledger.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/icrc-1/icrc1-ledger-setup.md:4]:
```suggestion
This guide will show you how to deploy an ICRC-1 ledger locally and on the mainnet, how to interact with the ICRC-1 ledger and how to test it. ICRC-1 is a token standard which you can read up on [here](https://github.com/dfinity/ICRC-1). Thus, deploying a new token that is ICRC-1 compatible is synonymous with deploying an ICRC-1 ledger. 
```

File: docs/developer-docs/integrations/icrc-1/index.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 # ICRC-1 token standard
 
 ## Overview
-The [ICRC-1](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md) is a standard for **fungible tokens** on the Internet Computer.
+
+ICRC-1 is a token standard created the internet computer working group. ICRC stands for "Internet Computer Request for Comments", you can find documentation on the working group [here](https://github.com/dfinity/ICRC). The [ICRC-1](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md) is a standard for **fungible tokens** on the Internet Computer. This means that any ICRC-1 ledger has to implement at least the specifications that are defined there.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/icrc-1/index.md:5]:
```suggestion
ICRC-1 is a token standard created by the Internet Computer working group. ICRC stands for "Internet Computer Request for Comments", you can find documentation on the working group [here](https://github.com/dfinity/ICRC). The [ICRC-1](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md) is a standard for **fungible tokens** on the Internet Computer. This means that any ICRC-1 ledger has to implement at least the specifications that are defined there. 
```

File: docs/developer-docs/integrations/icrc-1/index.md [hunk 3]
```diff
@@ -17,7 +20,7 @@ type Account = record { owner : principal; subaccount : opt Subaccount; };
 ```
 
 ## Methods
-
+This section will give you an overview of the methods of the ICRC-1 standard. If you want to have a more detailed description on how to interact with those enpoints you can have a look at [the guide on how to interact with an ICRC-1 ledger](./interact-with-ICRC-1-ledger.md)
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/icrc-1/index.md:23]:
```suggestion
This section will give you an overview of the ICRC-1 standard methods. If you want to have a more detailed description on how to interact with those endpoints, you can have a look at [the guide on how to interact with an ICRC-1 ledger](./interact-with-ICRC-1-ledger.md).
```

File: docs/developer-docs/integrations/ledger/index.md [hunk 4]
```diff
@@ -19,15 +19,15 @@ An account belongs to and is controlled by the account owner who must be an Inte
...
 
-An account on the ledger is identified by its address, which is derived from the principal ID and sub-account identifier.
+An account on the ledger is identified by its `AccountIdentifier`, which is derived from the principal ID and sub-account identifier.
 
 In this context, you can think of principal identifiers as a rough equivalent to the hash of a user’s public key for Bitcoin or Ethereum. You use the corresponding secret key to sign messages and therefore authenticate to the ledger canister and operate on the principal’s account. Canisters can also have accounts in the ledger canister, in which case the address is derived from the canister’s principal.
 
 The ledger canister is initialized using administrative operations that are internal to the Internet Computer. As part of the initialization process, the canister is created with the set of accounts and associated ICP token balances.
 
-:::info Why does the ledger use Account IDs and not just Principal IDs?
+:::info Why does the ledger use AccountIdentifiers and not just Principal IDs?
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ledger/index.md:28]:
```suggestion
:::info Why does the ledger use `AccountIdentifier`s and not just Principal IDs?
```

File: docs/developer-docs/integrations/ledger/index.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ The ICP token is similar to Bitcoin in the following ways:
 
 -   Tokens are entirely fungible.
 
--   Account identifiers are 32 bytes and are roughly the equivalent of the hash of a public key, optionally together with some additional sub-account specifier.
+-   `AccountIdentifiers` are 32 bytes and are roughly the equivalent of the hash of a public key, optionally together with some additional sub-account specifier.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ledger/index.md:62]:
```suggestion
-   `AccountIdentifier`s are 32 bytes and are roughly the equivalent of the hash of a public key, optionally together with some additional sub-account specifier.
```

File: docs/developer-docs/integrations/ledger/interact-with-ledger.md [hunk 6]
```diff
@@ -43,15 +62,188 @@ The transfer function can be used to transfer ICP from your account to another.
...
+export TO_ACCOUNT = "d52f7f2b7277f025bcaa5c90b10d122274faba2891bea519105309ae1f0af91d"
+dfx canister call ryjl3-tyaaa-aaaaa-aaaba-cai transfer '(record { to = $(python3 -c 'print("vec{" + ";".join([str(b) for b in bytes.fromhex("'$TO_ACCOUNT'")]) + "}")'); memo = 1:nat64; amount = record {e8s = 2_00_000_000 }; fee = record { e8s = 10_000 }; })'
+```
+
+This command returns the block index in which this transaction took place. In this example, it was the first block:
+```
+(variant { Ok = 1 : nat64 })
+```
+To get the balance of an `AccountIdentifier` you can call:
+dfx canister call ryjl3-tyaaa-aaaaa-aaaba-cai account_balance '(record { account = '$(python3 -c 'print("vec{" + ";".join([str(b) for b in bytes.fromhex("'$TO_ACCOUNT'")]) + "}")')' })'
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ledger/interact-with-ledger.md:141]: This line is missing the opening code block indicator ```

File: docs/developer-docs/integrations/ledger/introduction_and_overview.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+The Internet Computer utilizes two different types of ledgers. The first ledger is the ICP ledger. It is part of the [NNS](/tokenomics/nns/nns-intro.md) and the canister that implements
+the ICP ledger is running on the NNS subnet. The second type of ledger are the ICRC-1 ledgers. In this overview, both ledger types
+will be explained and their differences will be highlighted. 
+
+### ICP ledger
+The main purpose of the ICP ledger is to record `burn`, `mint` and most commonly `transfer` transactions with regards to the IC's native [token](/docs/concepts/tokens-cycles.md) `ICP`.
+The ICP ledger canister services transaction requests and offers a variety of endpoints to fetch data and information about the state of the ICP ledger.
+There is only one ICP ledger on the Internet Computer. 
+
+### ICRC-1 Ledgers
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/ledger/introduction_and_overview.md:14]:
```suggestion
### ICRC-1 ledgers
```

**Review by jessiemongeon1 — APPROVED** [review]

**Review by MarioDfinity — APPROVED** [review]

**Pull request merged**